.eslintrc.cjs

```diff
@@ -52,12 +36,16 @@ module.exports = {
     es2017: true,
     node: true,
   },
+  globals: {
+    App: 'readonly',
+    $$Generic: 'readonly',
+  },
   rules: {
     '@typescript-eslint/no-unused-vars': [
       'error',
       {
         argsIgnorePattern: '^_',
-        varsIgnorePattern: '^_',
+        varsIgnorePattern: '^_|^\\$\\$(Props|Events|Slots)$',
```

Comment:
This is needed for when there's a `(type|interface) $$Props = {}` (or `$$Events` or `$$Slots`) in a svelte component.

Reply:
https://github.com/sveltejs/eslint-plugin-svelte/issues/348